R/BiocWorkshopSubmit.R

```diff
@@ -101,18 +101,20 @@ BiocWorkshopSubmit <- function(...) {
                                 textInput("description", "Description"),
                                 textInput(
                                     "ghrepo",
-                                    label = mandatory("GitHub Repository"),
-                                    placeholder = "username/repository"
+                                    label = "Git Repository",
+                                    placeholder = "https://github.com/username/repository"
```

Comment:
Currently, the codebase only works for GitHub repositories. If you want to expand this to other `git` repo hosting platforms (GitLab, BitBucket, etc.), the codebase should support that. 
Changing the placeholder will affect the code that works with the `ghrepo` value, so that should change as well..